### PR TITLE
Add `page history as diffs` link

### PR DESCRIPTION
**Pull request by jasonyavorsky**
This adds a link which shows a sourcegraph view of what's changed on the page recently. This is a handy way, if you haven't visited a page in a while, to check what's new. It supplements rather than replaces the existing commit-based history view.

### CODE REVIEW

**Review by jeanduplessis — APPROVED** [review]
I think this is a great first iteration.

File: src/components/EditSection.tsx [hunk 1]
```diff
@@ -56,6 +56,12 @@ export const EditSection: React.FunctionComponent<EditSectionProps> = ({ page })
                 </div>
             )}
             <div className="sidebar-bottom-links">
+                <Link
+                    href={`https://sourcegraph.com/search?q=context:global+repo:%5Egithub.com/sourcegraph/handbook%24+file:${CONTENT_FOLDER}/${pagePath}+type:diff+rev:main&patternType=literal`}
+                >
+                    What's new as diff
```
Comment (jyavorska) [inline src/components/EditSection.tsx:62]:
```suggestion
                    See what's new (diffs)
```

Comment (jasonyavorsky) [inline src/components/EditSection.tsx]:
```suggestion
                    What's new as diffs
```

**Comment by quinnkeast** [comment]
+1, great first iteration of this.

File: src/components/EditSection.tsx [hunk 2]
```diff
@@ -56,6 +56,12 @@ export const EditSection: React.FunctionComponent<EditSectionProps> = ({ page })
                 </div>
             )}
             <div className="sidebar-bottom-links">
+                <Link
+                    href={`https://sourcegraph.com/search?q=context:global+repo:%5Egithub.com/sourcegraph/handbook%24+file:${CONTENT_FOLDER}/${pagePath}+type:diff+rev:main&patternType=literal`}
+                >
+                    What's new as diffs
```
Comment (jasonyavorsky) [inline src/components/EditSection.tsx]:
```suggestion
                    Page history as diffs
```

**Pull request merged**